### PR TITLE
feat(FEV-993): getting thumb cue points using cue point service

### PR DESCRIPTION
**Pull request by RoyBregman**
removing test temp code and registering to new cue point service

solves FEV-993

### CODE REVIEW

File: src/dualscreen.tsx [hunk 1]
```diff
@@ -387,6 +388,11 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
     }
   };
 
+  private _getThumbs() {
+    const kalturaCuePointService :any = this._player.getService('kalturaCuepoints');
+    kalturaCuePointService.registerTypes([kalturaCuePointService.CuepointType.SLIDE]);
```
Comment (yairans) [inline src/dualscreen.tsx:393]:
```suggestion
    kalturaCuePointService?.registerTypes([kalturaCuePointService.CuepointType.SLIDE]);
```

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/dualscreen.tsx (hunk 1 above)
Comment (RoyBregman) [reply src/dualscreen.tsx]: Maybe use "hasService" and warn if not?

Comment (yairans) [reply src/dualscreen.tsx]: yes

Comment (OrenMe) [reply src/dualscreen.tsx]: I think hasService is a nice addition to API in general, but for maybe other checks.
In this case we ask for service and if it doesn't exist then we just don't register types.
I would wait a bot with adding more API to services if not necessary until we can have better system to define service dependencies.

Comment (OrenMe) [reply src/dualscreen.tsx]: I think in the future when we do have this dependency management then in any case a plugin can wait for a service and require it only once in the constructor or init phase of the plugin and in this case we can handle lack of service a bit more gracefully.
@RoyBregman this is related to the discussion we had about finding good reasons to have this in place.

**Review by yairans — APPROVED** [review]

**Review by semarche-kaltura — APPROVED** [review]

**Pull request merged**